website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -60,6 +61,28 @@ For example, given the forward-to URL `https://my-custom-app.com/api/123`, the `
 - `forward-url-full-path`: `https://my-custom-app.com/api/123`
 - `forward-url-origin`: `https://my-custom-app.com`
 
+## Configuring custom claims
+
+<Info>
+
+This feature is available from version `21.14.0` onwards.
```

Comment:
This will need to be adjusted once we know the actual version where we will release the feature.

Reply:
I moved it beneath the supported claims list.
Perhaps we can leave it like this and change it in the future, when we'll support more claims, to be more specific enumerating which version support which claim.

Updated in this commit: 088fb79b95c5c8d8d309617db4f8f9f1a47da13b